socorro/external/es/crashstorage.py

```diff
@@ -17,7 +17,10 @@
 
 
 # Maximum size in characters for a keyword field value
-MAX_KEYWORD_FIELD_VALUE_SIZE = 10000
+MAX_KEYWORD_FIELD_VALUE_SIZE = 10_000
+
+# Maximum size in utf-8 encoded characters for a string field value
+MAX_STRING_FIELD_VALUE_SIZE = 32_766
```

Comment:
This number comes from Elasticsearch. When we upgrade to a newer version of Elasticsearch, we can see what the maximum size for a term is and adjust this accordingly.